onsite/phospho_scoring.py

```
@@ -0,0 +1,643 @@
+                        phospho_count = len([h for h in result['new_pid'].getHits() 
+                                          if "(Phospho)" in h.getSequence().toString()])
+                        stats['phospho'] += phospho_count
+                        
+                        if logger and logger.isEnabledFor(logging.INFO):
+                            logger.info(f"Successfully processed identification {i+1} with {phospho_count} phosphorylated hits")
+                    else:
+                        stats['errors'] += 1
+                        error_msg = f"Error processing identification: {result['reason']}"
+                        if logger and logger.isEnabledFor(logging.ERROR):
+                            logger.error(error_msg)
+                except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)